### ISSUE TITLE
Language server crashes while prasing dependencies

### ISSUE THREAD

**Issue by Karolis2011** [felixfbecker/php-language-server#133]
Language server fails to prase code with fully populated dependencies of Symfony framework.

**Operating System:** Windows 10 64-bit [_10.0.14393_]
**Visual Studio configuration:** 
Version _1.7.1_
**PHP InteliSense extension version:** _0.0.11_
**PHP version:** _7.0.12_

**Symfony version:** _3.1.6_

Language server last lines were:
```
...
[Info  - 14:37:02] Parsing file 114/4736: var\SymfonyRequirements.php.
[Info  - 14:37:02] Parsing file 115/4736: vendor\autoload.php.
[Error - 14:37:02] Connection to server got closed. Server will not be restarted.

```
Exception:
```
PHP Fatal error:  Uncaught Symfony\Component\Debug\Exception\ContextErrorException: Warning: filesize(): stat failed for c:\Users\karol\Desktop\projects\projectname\vendor\bin\doctrine.php in C:\Users\karol\.vscode\extensions\felixfbecker.php-intellisense-0.0.11\vendor\felixfbecker\language-server\src\LanguageServer.php:169
Stack trace:
#0 [internal function]: Symfony\Component\Debug\ErrorHandler->handleError(2, 'filesize(): sta...', 'C:\\Users\\karol\\...', 169, Array)
#1 C:\Users\karol\.vscode\extensions\felixfbecker.php-intellisense-0.0.11\vendor\felixfbecker\language-server\src\LanguageServer.php(169): filesize('c:\\Users\\karol\\...')
#2 C:\Users\karol\.vscode\extensions\felixfbecker.php-intellisense-0.0.11\vendor\sabre\event\lib\Loop\Loop.php(280): LanguageServer\LanguageServer->LanguageServer\{closure}()
#3 C:\Users\karol\.vscode\extensions\felixfbecker.php-intellisense-0.0.11\vendor\sabre\event\lib\Loop\Loop.php(216): Sabre\Event\Loop\Loop->runTimers()
#4 C:\Users\karol\.vscode\extensions\felixfbecker.php-intellisens in C:\Users\karol\.vscode\extensions\felixfbecker.php-intellisense-0.0.11\vendor\felixfbecker\language-server\src\LanguageServer.php on line 169
```

**Comment by felixfbecker** [comment]
But why did the stat fail... Maybe the file is locked? Maybe try to disable antivirus

**Comment by Karolis2011** [comment]
So, solution wasn't disabled Antivirus (I am using Windows Defender). I realized that it was problem with Ubuntu subsystem for Windows 10.

So I locked file reminded me of time when I tried to copy whole populated project to USB memory stick and Windows failed to do so, because it failed to read files. I had that error with other files created in Ubuntu subsystem. So this time i just deleted `vendor` folder and repopulated it using composer from Windows instead of Ubuntu bash and Inteli Sense features seems to work. This is problem with  Ubuntu subsystem for Windows 10. This issue is resolved.
